spec/support/shared_examples/pattern.rb

```diff
@@ -8,7 +8,7 @@
   end
 
   it { expect(pattern.structure).to be_name name }
-  it { check_binary_file_content(file_name, fixtures_path("generated_patterns/#{name}.svg")) }
+  it { expect(file_name).to have_same_file_content_like(fixtures_path("generated_patterns/#{name}.svg")) }
```

Comment:
@jasonlong Please try something like

~~~ruby
it { expect(file_name).to have_same_file_content_like("%/generated_patterns/#{name}.svg") }
~~~

`%` is [expanded](https://github.com/cucumber/aruba/blob/master/lib/aruba/matchers/file/have_same_file_content.rb#L32) to `<project_root>/fixtures`.